docker/Dockerfile

```diff
@@ -51,16 +51,21 @@ WORKDIR /dav1d/build
 
 RUN meson setup ../; \ 
     ninja; \
-    ninja install 
+    ninja install; \
+    ln -s /usr/local/lib/x86_64-linux-gnu/libdav1d.so.7.0.0 /usr/local/lib/libdav1d.so.7; \
+    ln -s /usr/local/lib/x86_64-linux-gnu/libdav1d.so.7.0.0 /usr/local/lib/libdav1d.so.7.0; \
+    ln -s /usr/local/lib/x86_64-linux-gnu/libdav1d.so.7.0.0 /usr/local/lib/libdav1d.so.7.0.0; \
+    echo "/usr/local/lib" >> /etc/ld.so.conf.d/mylibs.conf \
+    && ldconfig
 
 #Cargo build for stump
 WORKDIR /app
 
 COPY . .
 
-RUN ./scripts/release/utils.sh -w; \
+RUN cargo prisma generate --schema ./core/prisma/schema.prisma; \
+    ./scripts/release/utils.sh -w; \
     set -ex; \
-    ./scripts/release/utils.sh -p; \
```

Comment:
Note that I actually had to remove the `prisma.rs` text replacements because it was actually generated in the `Dockerfile` above so was already correct